src/cascadia/TerminalSettingsEditor/ColorSchemes.xaml

```diff
@@ -194,6 +264,23 @@ the MIT License. See LICENSE in the project root for license information. -->
                     <TextBlock x:Uid="ColorScheme_SelectionBackground"
                                Style="{StaticResource ColorHeaderStyle}"/>
                     <Button Background="{Binding Color, ElementName=SelectionBackgroundPicker, Converter={StaticResource ColorToBrushConverter}, Mode=OneWay}">
+
+
+                        <Button.Resources>

```

Comment:
Grrr... Annoying that we have to do this on every button here. But I understand why :(

Reply:
Yea, I don't think I can use a template to have all these set automagically. I might be wrong on that, but it certainly doesn't seem like I can use a `Setter` to set the `Resources` like this